torch/onnx/_internal/diagnostics/infra/sarif_om/__init__.py

```diff
@@ -0,0 +1,108 @@
+# DO NOT EDIT! This file was generated by jschema_to_python version 0.0.1.dev29,
+# with extension for dataclasses and type annotation.
+
```

Comment:
How about moving all generated files to a `generated` folder? That possibly helps in automating "cleaning builds" during development

Reply:
I am guessing `sarif_om` stands for `sarif_object_model`, but being more explicit in the naming makes things easier for first timers

Reply:
How about `sarif` then